packages/terra-form-textarea/src/Textarea.scss

```diff
@@ -0,0 +1,151 @@
+:local {
+  .textarea {
+    background-color: var(--terra-form-textarea-background-color, #ffffff);
+    background-image: var(--terra-form-textarea-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-textarea-focus-background-start, 0 0);
+    border-color: var(--terra-form-textarea-border-color, #dedfe0);
+    border-radius: var(--terra-form-textarea-border-radius, 3px);
+    border-style: var(--terra-form-textarea-border-style, solid);
+    border-width: var(--terra-form-textarea-border-width, 1px);
+    color: var(--terra-form-textarea-color, #000);
+    cursor: var(--terra-form-textarea-cursor, auto);
+    font-family: var(--terra-base-font-family, 'Helvetica Neue', Helvetica, Arial, sans-serif);
+    font-size: var(--terra-form-textarea-font-size, 1.143rem);
+    line-height: var(--terra-form-textarea-line-height, 1.25);
+    opacity: var(--terra-form-textarea-focus-opacity, 1);
+    padding-bottom: var(--terra-form-textarea-padding-bottom, 0.286rem);
+    padding-left: var(--terra-form-textarea-padding-left, 0.286rem);
+    padding-right: var(--terra-form-textarea-padding-right, 0.286rem);
+    padding-top: var(--terra-form-textarea-padding-top, 0.286rem);
+    transition-duration: var(--terra-form-textarea-focus-transition-duration, 0.2s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-textarea-focus-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border-color: var(--terra-form-textarea-hover-border-color, #dedfe0);
+      border-style: var(--terra-form-textarea-hover-border-style, solid);
+      border-width: var(--terra-form-textarea-hover-border-width, 1px);
+    }
+
+    &[required] {
+      box-shadow: unset;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-textarea-disabled-background-color, rgba(111, 116, 119, 0.05));
+      border-color: var(--terra-form-textarea-disabled-border-color, #dedfe0);
+      border-style: var(--terra-form-textarea-disabled-border-style, solid);
+      border-width: var(--terra-form-textarea-disabled-border-width, 1px);
+      color: var(--terra-form-textarea-disabled-color, rgba(0, 0, 0, 0.25));
+      opacity: var(--terra-form-textarea-disabled-opacity, 1);
+      resize: none;
+    }
+
+    &:focus {
```

Comment:
Based on [this finding](https://github.com/cerner/terra-core/issues/972#issuecomment-341928670) @JakeLaCombe, I want to remove the mouse only focus styles and make the keyboard focus styles the default focus styles for this PR. Once we land a solution to determine focus source, we can re-add the different focus styles back to textarea in a future PR.

Reply:
Done. https://github.com/cerner/terra-core/pull/982/commits/9893550863f5a1367263a25c8b357aa79768bb39